crates/bevy_pbr/src/alpha.rs

```diff
@@ -23,6 +23,30 @@ pub enum AlphaMode {
     /// Standard alpha-blending is used to blend the fragment's color
     /// with the color behind it.
     Blend,
+    /// Similar to [`AlphaMode::Blend`], however assumes RGB channel values are
+    /// [premultiplied](https://en.wikipedia.org/wiki/Alpha_compositing#Straight_versus_premultiplied).
+    ///
+    /// For fixed RGB values, behaves more like [`AlphaMode::Blend`] for
```

Comment:
What do you mean by fixed RGB values?

Reply:
Hmm yeah that came out confusing. Changed it to “For otherwise constant RGB values...”